test/new-e2e/tests/agent-platform/common/agent_behaviour.go

```diff
@@ -192,23 +192,34 @@ func CheckDogstatsdAgentRestarts(t *testing.T, client *TestClient) {
 	})
 }
 
-// CheckAgentPython runs tests to check the agent use the correct python version
-func CheckAgentPython(t *testing.T, client *TestClient, version string) {
-	t.Run(fmt.Sprintf("set python version %s and restarts", version), func(tt *testing.T) {
+const (
+	// ExpectedPythonVersion2 is the expected python 2 version
+	// Bump this version when the version in omnibus/config/software/python2.rb changes
+	ExpectedPythonVersion2 = "2.7.18"
+	// ExpectedPythonVersion3 is the expected python 3 version
+	// Bump this version when the version in omnibus/config/software/python3.rb changes
+	ExpectedPythonVersion3 = "3.11.5"
```

Comment:
If we cannot read directly the value from the omnibus config, should we add a (pre) check to assess this (that expected versions are aligned)?

Reply:
Is there a job I could use ? Or better to add a new github action ?

Reply:
Thinking about it, there are more spots where the python version is updated. If we want to automate that we don't miss anything, we should address this separately - see latest bump PR for an example https://github.com/DataDog/datadog-agent/pull/20218

Reply:
Tracked by APL-2814